lib/routes/tencent/cloud/developer/column.ts

```diff
@@ -0,0 +1,78 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { parseDate } from '@/utils/parse-date';
+
+const PAGE = 1;
+const PAGE_SIZE = 20;
+
+export const route: Route = {
+    path: '/cloud/developer/column/:categoryId?',
+    categories: ['social-media', 'programming'],
+    example: '/tencent/cloud/developer/column/1',
+    parameters: { categoryId: '专栏Id: categoryId == classifyId(api)' },
```

Comment:
Please update the description in a way that an average person can understand.

Reply:
done